src/types/ERC725JSONSchema.ts

```diff
@@ -31,6 +32,7 @@ export type ERC725JSONSchemaValueType =
   | 'bytes32[]'
   | 'bytes4[]'
   | 'bytes[]'
+  | 'boolean'
```

Comment:
so after the discussions, this should be `bool` not `boolean` ?

Reply:
or both as you said you want to support both

Reply:
@Hugoo yes should support both in the library: `bool` and `boolean`. Added the types